### PR TITLE
[change] simplify mix.exs versioning due to how deploy scripts work

### PR DESCRIPTION
**Pull request by zoten**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -4,7 +4,7 @@ defmodule RedactEx.MixProject do
   def project do
     [
       app: :redact_ex,
-      version: get_version(System.get_env("DRONE_TAG")),
+      version: "0.1.3",
```
Comment (WilliamVenner) [inline mix.exs:7]: could this benefit from a [`.versions.yml`](https://github.com/primait/suite_py/blob/master/docs/index.md#i-want-to-bump-the-project-version) for `suite-py bump`?

Comment (zoten) [reply mix.exs:7]: didn't know about it! However I don't see it used in other prima public libraries, am I wrong?

Comment (zoten) [reply mix.exs:7]: (looking at amqpx, prima_ex_logger)

Comment (WilliamVenner) [reply mix.exs:7]: I guess they just haven't adopted it yet - I just [added it to Veil](https://github.com/primait/veil/pull/41/commits/4bd65434c306e7683a9d71167e0f9ec97f33ae8b)

Comment (zoten) [reply mix.exs:7]: nice, added! thanks!

**Review by WilliamVenner — APPROVED** [review]

**Pull request merged**